examples/Backtesting/botorch_analytical.py

```diff
@@ -124,8 +129,15 @@
     n_mc_iterations=N_MC_ITERATIONS,
 )
 
-# The following lines plot the results and save the plot in run_analytical.png
-
-sns.lineplot(data=results, x="Num_Experiments", y="Target_CumBest", hue="Scenario")
-plt.gcf().set_size_inches(24, 8)
-plt.savefig("./run_analytical.png")
+themes = json.load(open("plotting_themes.json"))
+for theme in themes:
+    sns.set_theme(style="ticks", font_scale=1.75, rc=themes[theme])
```

Comment:
can't the font scaling be included in the style file?

Reply:
Might be possible via one additional layer of nesting, i.e., have each theme on the top level contain two sublevels with font_scale and the rc_params. Will double check.

Reply:
Included it in the style file.